style.css

```diff
@@ -207,4 +207,232 @@ body {
 .container a:hover {
   background-color: bisque;
   color: black;
+}
+
+ul {
+  display: grid;
+  grid-template-columns: repeat(3, 40px);
+  gap: 0;
+  margin: 5px 0;
+  list-style: none;
+}
+
+.skills {
+  text-decoration: none;
+  background-color: #ebebff;
+  text-align: center;
+  padding: 5px 7px;
+  border-radius: 10px;
+  font-size: 13px;
+}
+
+/* stylelint-disable-next-line no-duplicate-selectors */
+.container-section {
+  background: #e5e5e5;
+  width: 350px;
+  border-radius: 20px;
+}
+
+.grid-container-card {
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  gap: 70px;
+  margin-bottom: 10px;
+}
+
+.grid-card {
+  background-color: white;
+  padding: 15px 5px;
+  margin: 0 20px;
+  border-radius: 10px;
+}
+
+.responsive-backgroundImage {
+  height: 200px;
+  width: 300px;
+  margin: 0 auto;
+}
+
+.responsive-backgroundImage img {
+  height: 100%;
+  width: 100%;
+}
+
+.identification {
+  margin: 15px 10px;
+  font-family: sans-serif;
+}
+
+.identification h3 {
+  font-weight: 600;
+  font-size: 30px;
+  margin: 10px 0;
+}
+
+.flex-container {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  font-weight: 800;
+}
+
+.flex-container .title-bolder {
+  font-size: 11px;
+  margin: 10px 0;
+}
+
+.flex-container .title-non-bolder {
+  font-size: 11px;
+  margin: 10px 0;
+  color: rgb(148, 147, 147);
+}
+
+.flex-container .point {
+  width: 5px;
+  height: 5px;
+  background-color: gray;
+  border-radius: 50%;
+  margin: 0 15px;
+  color: rgb(148, 147, 147);
+}
+
+.text p {
+  margin: 5px 0;
+  color: rgb(63, 64, 62);
+}
+
+/* Adding padding to the button. */
+.btn {
+  padding: 8px;
+  border-radius: 5px;
+  margin: 8px 0;
+  transition: all 0.5s;
+  border: 2px solid #396df2;
+  color: #396df2;
+  font-size: 18px;
+  font-weight: 500;
+}
+
+.btn:hover {
+  background: rgb(14, 12, 12);
+  color: beige;
```

Comment:
Kindly make sure the background color of the enabled button is not the button default color please change it to your design

please go to this section of your Figma design to select the exact color of the button for enabled and hover

![image](https://user-images.githubusercontent.com/43564324/159125083-4ceb9b25-ad83-4294-ab3f-4a10584d7d7e.png)